### DISCUSSION TITLE
How to stop going to next difficulty?

### DISCUSSION THREAD

**Discussion by Outlaw466** [isid0re/SoloLeveling#340] [Q&A]
How do I change/override starting the new Difficulty?
I want it to stay farming nightmare til 80.
I have a BlizzBaller and it has the resistances to go to hell so the bot is making it go to hell and it's just getting destroyed.

**Comment by tyson-dev** [comment]
kolbot-master\d2bs\kolbot\libs\SoloLeveling\Functions > globals.js > line 56

"levelCap: [33, 65, 100][me.diff],"

change the 65 to 80 ( keep in mind if you still have negative res he will try to go until 85 then before forcing the switch to hell diff)
